### ISSUE TITLE
Stopped working after months (Screenshot of 3dmigoto shows pyinjector error)

### ISSUE THREAD

**Issue by DraGZ76** [SilentNightSound/SR-Model-Importer#46]
![hmeJG1L](https://github.com/SilentNightSound/SR-Model-Importer/assets/82879008/3d0829e7-63ed-44b6-ba13-d8b4ec25f91a)

**Comment by CzHUV** [comment]
Try run it as Administrator and launch sr through Mihoyo Launcher instead of directly starting the _.exe file_.

**Comment by DraGZ76** [comment]
> Try run it as Administrator and launch sr through Mihoyo Launcher instead of directly starting the _.exe file_.

Yeah I'm never launching through .exe in my file I have path to launcher
+ I'm always on admin profile so I don't need to change permissions or anything

**Comment by CzHUV** [comment]
Sorry for not helping :(
But I did make it work through this way,maybe you can DM the creator and see if he could do anything with that
Hope your problem will be solved soon:)

**Comment by Thesuspie** [comment]
i have fixed the issue, by installing the dependency and recompiling it, the bigger problem is that for some reason cmd would refuse run it correctly so i've made a script that changes the shell. find the instructions in the README.md and the download in the releases of the fork that i made as a hotfix for now. link: https://github.com/Thesuspie/SR-Model-Importer

**Comment by mimmox** [comment]
Same problem, I have been using it for 3 months without any problems, since the day before yesterday I don't know why it started having problems.
